api/db/changelog/db.changelog-235-replace-workspace-billing-status.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="evrii" id="db.changelog-234-replace-workspace-billing-status">
+    <addColumn tableName="workspace">
+      <column name="initial_credits_exhausted" type="boolean" defaultValueBoolean="false">
+        <constraints nullable="false"/>
+      </column>
+      <column name="initial_credits_expired" type="boolean" defaultValueBoolean="false">
+        <constraints nullable="false"/>
+      </column>
+    </addColumn>
+    <sql>
```

Comment:
All workspaces in BillingStatus.INACTIVE should be because of initial credit use, because BillingStatus does not track user provided billing account usage.

Setting the initial values takes care of current state. Elsewhere, all updates to BillingStatus are accompanied by an update to either `initial_credits_exhausted` or `initial_credits_expired` to keep the two fields in sync.